cidemipyme-gatsby/src/components/sections/ContactSection.jsx

```diff
@@ -131,6 +161,7 @@ class ContactSection extends React.Component {
           {messageFeedback}
         </FormGroup>
         {sentMessage}
+        {sentFailureMessage}
         <Button
```

Comment:
remove the type `submit` with that you can avoid `event.preventDefault();` and you do no longer need to send the event, so you can remove the parameter